test/support/test_config.h

```diff
@@ -76,7 +76,9 @@
 #endif
 
 // Enable test when the TBB backend is available
-#define TEST_TBB_BACKEND_PRESENT (!defined(ONEDPL_USE_TBB_BACKEND) || ONEDPL_USE_TBB_BACKEND != 0)
+#if !defined(ONEDPL_USE_TBB_BACKEND) || ONEDPL_USE_TBB_BACKEND

```

Comment:
Don't understand this one.

Reply:
Because otherwise we had the next warning when use this macros in code:
```
test/support/test_config.h - Warning G2FB9CE64 macro expansion producing 'defined' has undefined behavior [-Wexpansion-to-defined]
```